app/Observers/EntryObserver.php

```diff
@@ -28,11 +29,13 @@ public function created(Entry $entry)
      */
     public function updated(Entry $entry)
```

Comment:
Just to be clear, this observer assumes that we cannot detach a DM Entry once we've attached it to a character, right? Correct me if I'm wrong, but if we were to remove a `character_id` from an entry, it would not substract the levels from the characters.

Reply:
Once an entry is added to a character, it should be deleted, not removed.

Upon deletion, the characters level will be updated to reflect the change